packages/sdk/src/controllers/VariableController.ts

```diff
@@ -480,23 +481,72 @@ export class VariableController {
     };
 
     /**
+     * @deprecated the visibility setter is not supported anymore.
+     * Use `setVariableVisibility` instead.
      * This method sets isVisible flag for a variable
+     * @param _id variable id
+     * @param _isVisible visibility flag
      * @returns
      */
-    setIsVisible = async (id: string, isVisible: boolean) => {
-        const res = await this.#editorAPI;
-        return res.setVariableIsVisible(id, isVisible).then((result) => getEditorResponseData<null>(result));
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    setIsVisible = async (_id: string, _isVisible: boolean) => {
```

Comment:
Perhaps we deprecate the method but have it route to the setVisibility for now. WDYT @brapoprod